src/inputs/Button/Button.tsx

```diff
@@ -1,44 +1,55 @@
-import { PropsWithChildren } from "react";
-import * as React from "react";
-
+import React from "react";
 import styles from "./Button.module.css";
+import { Spinner } from "../../icons";
+import { Plus } from "../../icons";
+import { Fragment } from "react";
+import classNames from "classnames/bind";
+const cx = classNames.bind(styles);
 
-export interface ButtonProps {
-  onClick(): void;
+interface ButtonProps {
+  type?: "button" | "submit";
   disabled?: boolean;
-  variant?: "primary" | "secondary";
-  isLoading?: boolean;
+  isLoading: boolean;
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  variant: "primary" | "secondary";
+  children: React.ReactNode;
```

Comment:
**nitpick:** We should not define children this way. Use `PropsWitchChildren<ButtonProps>` instead